js/chat_app/src/components/LogOut/LogOut.jsx

```diff
@@ -0,0 +1,118 @@
+import AppBar from '@mui/material/AppBar';
+import Button from '@mui/material/Button';
+import { deepOrange } from '@mui/material/colors';
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import authSelectors from '../../redux/auth/authSelector';
+import Container from '@mui/material/Container';
+import Toolbar from '@mui/material/Toolbar';
+import Avatar from '@mui/material/Avatar';
+import Typography from '@mui/material/Typography';
+import Dialog from '@mui/material/Dialog';
+import DialogContent from '@mui/material/DialogContent';
+import DialogActions from '@mui/material/DialogActions';
+import Stack from '@mui/material/Stack';
+import Badge from '@mui/material/Badge';
+import { createTheme, styled } from '@mui/material';
+import { ThemeProvider } from '@emotion/react';
+import { isLogout } from '../../redux/auth/authSlice';
+
+function LogOut({ username, password }) {
+  const defaultMode = true;
+  const [mode, setMode] = useState(defaultMode);
+  const [isWindowLogoutOpen, setIsWindowLogoutOpen] = useState(false);
+  const name = useSelector(authSelectors.getUsername);
+  const isLogin = useSelector(authSelectors.getLogin);
+  const dispatch = useDispatch();
+  const navigation = useNavigate();
+  const theme = createTheme({
```

Comment:
i don't think that Logout it's a good place to use the theme like that. It's better to use Context or Redux in that case. So every component will be able to get changes.